### DISCUSSION TITLE
Regarding Starting and testing our code change in local

### DISCUSSION THREAD

**Discussion by Mohit269** [agershun/alasql#1680] [General]
I just clone this repo will anyone guide me how to start and test this in my local vs code.

**Comment by mathiasrw** [comment]
Sure - and sorry for super slow response. 

From https://github.com/AlaSQL/alasql/blob/develop/CONTRIBUTING.md 	


> For you to submit a pull request: 
> 
> - Make sure you have git, Node and yarn installed (`npm install -g yarn`)
> - Fork the repo here on Github (button top right)
> - Clone your forked repo and install dependencies `git clone https://github.com/MYUSERNAME/alasql/ --depth 1 && cd alasql && yarn` 
> - Make sure you work with the develop branch `git checkout develop`
> - Make sure you got dependencies installed `yarn`
> - Run tests to verify all is good `yarn test`
> - Implement a test that reflects the issue.
>   - Add a new test file for the issue: Copy `test/test000.js` and replace `000` with a new number. Preferably the number of the issue you are solving.
> - Run `yarn test` to verify only the new test fails
> - Implement your contributions in `src/`
> - Run `yarn test` and verify all tests are OK
> - Format the souce with `yarn format`
> - Commit changes to git and push to your forked repo
> - Click "Create Pull-request" when looking at your forked repo on Github

So in short: Make sure you got yarn installed, run yarn install then you can run the tests via 'yarn test'
